### ISSUE TITLE
Low contrast hyperlinks in fltk.org

### ISSUE THREAD

**Issue by rodarima** [fltk/fltk#1474]
Hi, this is a problem related to fltk.org, not the FLTK library.

The tables in fltk.org have hyperlinks in red (`#cc2020`) in a grey-sh background (`#aaaadd` and `#9999cc`), which have low contrast ratio (2.5:1 and 2.05:1 respectively). In my particular case, I use a red filter to reduce blue light so the linked text become barely readable. I often need to select the text to read it or disable CSS.

I attach a picture of the screen taken from <https://www.fltk.org/newsgroups.php?gfltk.bugs+T> so you can see:

<img width="925" height="684" alt="Image" src="https://github.com/user-attachments/assets/1392ccc2-bfd1-442a-84e8-a0434885ae7d" />

This choice of colors is currently failing both AAA and AA levels in [WCAG](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html). My suggestion is that you adjust the colors to at least pass the AA level in WCAG:

https://webaim.org/resources/contrastchecker/

If you want keep the red theme, you can do that by making the link color darker and the background lighter, for example this pair increases the contrast ratio to 6:1 and passes both AA and AAA for large text:

<img width="776" height="823" alt="Image" src="https://github.com/user-attachments/assets/ec3acf10-7099-40b8-aace-1562ede6c541" />
